src/components/App/__test__/App.component.test.jsx

```diff
@@ -0,0 +1,34 @@
+import { render, cleanup, screen } from '@testing-library/react';
+import App from '../';
+import React from 'react';
+
+afterEach(cleanup);
+beforeEach(() => render(<App />));
+
+describe('header', () => {
+    test('header renders', () => {
+        const header = screen.queryByRole('navigation');
+        expect(header).toBeInTheDocument();
+    });
+
+    test('header elements render', () => {
+        const navButton = screen.queryByRole('button');
```

Comment:
Let us try to not use `queryBy` selectors unless testing for things that are not in the document per [Common Mistakes with React Testing Library](https://kentcdodds.com/blog/common-mistakes-with-react-testing-library#using-query-variants-for-anything-except-checking-for-non-existence)

Both `getBy` and `findBy` are better alternatives depending on your needs

Reply:
Ok, I will try to use getBy over queryBy from now on. Thanks.